### PR TITLE
Add $fixture_file to the fixtures docs

### PR DESCRIPTION
**Pull request by andrewandante**
Currently no mention of it.

### CODE REVIEW

**Comment by dhensby** [comment]
Can you cover using an array of files, please?

Also squash :)

**Comment by andrewandante** [comment]
Updated @dhensby

**Comment by dhensby** [comment]
Behat tests seem to be playing up at the moment :/

File: docs/en/02_Developer_Guides/06_Testing/04_Fixtures.md [hunk 1]
```diff
@@ -9,6 +9,42 @@ SilverStripe starts with a fresh database containing no records. `Fixtures` prov
 to load into the database. The [api:SapphireTest] class takes care of populating a test database with data from
 fixtures - all we have to do is define them.
 
+<div class="info" markdown='1'>
```
Comment (dhensby) [inline docs/en/02_Developer_Guides/06_Testing/04_Fixtures.md]: Hmm will wrapping all this in a markdown div act properly?

Comment (andrewandante) [inline docs/en/02_Developer_Guides/06_Testing/04_Fixtures.md]: No idea. Also no idea how to tell...

**Pull request merged**